packages/gasket-engine/README.md

```diff
@@ -25,34 +25,34 @@ The `GasketAPI` object passed to hook functions has the following members:
 
 The Gasket configuration object, typically derived from `gasket.config.js`.
 
-### exec(event, ...args)
+### `exec(event, ...args)`
```

Comment:
Per [contributing guide](https://github.com/godaddy/gasket/blob/main/CONTRIBUTING.md#markdown-documentation)
> - Never use `inline code` in headings

I recall we were encountering hash link strangeness, either with GitHub or Docsify (or both). It does not appear to be an issue with GH at least now. But, any particular reason why this change? Maybe it's fine? But if so, we need to update the contributing guide if we want to move to this format.

Reply:
The reason for this, unfortunately, is MDX parsing the file and executing these lines as code. Using the tick allows for the MDX so see it as markdown instead of actual code which results in errors.

Reply:
I feel like the contributing guide could remain? This is kind of a one off case IMO. I agree with the rule overall this is just what I had to do to resolve the errors.